src/CommunityToolkit.Maui.Core/Views/PaddedButton.macios.cs

```diff
@@ -18,7 +18,18 @@ public PaddedButton(nfloat leftPadding, nfloat topPadding, nfloat rightPadding,
 		RightPadding = rightPadding;
 		BottomPadding = bottomPadding;
 
+#if IOS15_0_OR_GREATER || MACCATALYST15_0_OR_GREATER
```

Comment:
I haven't found the right way to fix the warning except for pragma.
![image](https://user-images.githubusercontent.com/33021114/158614655-9e9d8e25-cfd7-4d8d-9320-e808ff1a04ab.png)

Reply:
NOT RELATED TO PREVIEW 14, but fixes issue on Windows 17763